### ISSUE TITLE
adding toctoc

### ISSUE THREAD

**Issue by tiagomoraismorgado88** [mozilla/web-ext#1163]
i think it would be good to:
- add toctoc node module via npm

**Comment by kumar303** [comment]
I agree. We have an example for a solution in  https://github.com/mozilla/web-ext/blob/master/CONTRIBUTING.md but it's not perfect.
